### PR TITLE
fix(backend): Add 'CSRF_TRUSTED_ORIGINS' django setting

### PR DESCRIPTION
**Pull request by wkyoshida**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

- Add `CSRF_TRUSTED_ORIGINS` django setting
   - Needed in addition to `ALLOWED_HOSTS` so `POST` requests are not blocked ([ref](https://www.reddit.com/r/django/comments/s6daj0/csrf_verification_failed_django_nginx_docker/))
- Add command `python manage.py collectstatic --no-input` already to the `deploy.Dockerfile` so Docker image already has static files prepped 

### Related issue

- :rocket:

### CODE REVIEW

File: backend/Dockerfile
Comment (andrewtavis) [inline backend/Dockerfile:3]: ❤️

**Review by andrewtavis — APPROVED** [review]
I'll let @to-sta take a look before merging, but fine on my end :) Thanks, @wkyoshida!

**Comment by to-sta** [comment]
👌

**Pull request merged**
